docs/sdk/client-side-sdks/react-native/react-native-install.md

```diff
@@ -0,0 +1,117 @@
+---
+title: React Native SDK Installation
+sidebar_label: Installation
+sidebar_position: 1
+---
+
+## Installation
```

Comment:
Might want to remove this as a double header? You call on if it looks weird or not.